engine/src/main/java/org/camunda/bpm/engine/impl/util/ParseUtil.java

```diff
@@ -108,7 +108,11 @@ public static ProcessEngineDetails parseProcessEngineVersion(String packageImple
     String version = packageImplementationVersion;
```

Comment:
Why don't we drop the packageImplementationVersion in general? That way the version will be fetch all that time only from the properties file and it will be consistent.

Reply:
I created an additional method so that the tests in `ProcessEngineDetailsTest` class remain valid and able to test all the possible use-cases.